### PR TITLE
Update Jupyter Quickstart

### LINKED ISSUE #1616: [Feature Request]: remove OdhApplication file and update getting started doc links in OdhQuickStarts
### Feature description

Related to term and usage of Jupyterhub in Dashboard

https://github.com/opendatahub-io/odh-dashboard/issues/534

The content itself in the OdhQuickstart documentation (not the getting started links) has been improved recently, i.e. no more references to installing something via npm and conda and so on for the most part, good on odh-manifests and here

https://github.com/opendatahub-io/odh-dashboard/tree/main/manifests/apps/jupyter

https://github.com/opendatahub-io/odh-manifests/tree/master/odh-dashboard/apps/jupyter

**Exept** in jupyterhub app

https://github.com/opendatahub-io/odh-manifests/blob/master/odh-dashboard/apps/jupyter/jupyterhub-app.yaml

there is talk of npm and all that.

But, the jupyter tile will be removed at some point anyways, so then, the following OdhApplication file jupyterhub-app.yaml can be removed in the process and shall not be linked to from anywhere anymore in Dashboard, which is done via #534 

In all files remaining, there is still a getting started link to jupyterhub docs, which seems wrong.

@MelissaFlinn Can you replace the `getStartedLink` in all remaining ODHQuickstarts with 

https://opendatahub.io/docs/ 

or https://opendatahub.io/docs/getting-started-with-open-data-hub/#creating-a-project-workbench_get-started

instead of the very Jupyterhub specific link

and change the `getStartedMarkDown` to something more in line with the content of https://opendatahub.io/docs/getting-started-with-open-data-hub/#launching-jupyter-and-starting-a-notebook-server_get-started

i.e. at https://github.com/opendatahub-io/odh-manifests/blob/master/odh-dashboard/apps/jupyter/create-jupyter-notebook-quickstart.yaml

please? Describing 

https://github.com/opendatahub-io/odh-dashboard/tree/main/manifests/apps/jupyter

At any files except jupyterhub-app.yaml basically, since @DaoDaoNoCode already removes that file in a separate PR :-)

There are for example currently instructions like " Locate the **Jupyter** card on the **Enabled applications** page".
As discussed with @lucferbux, that point of entry for a getting started, the jupyter card, can stay for now.

### Describe alternatives you've considered

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by lucferbux**
## Description

Closes #1616 

Follow up #1636 to get automation right.
Update the quickstarts to remove JupyterHub

Screenshot 2023-09-01 at 13 43 30
Screenshot 2023-09-01 at 13 40 30

## How Has This Been Tested?

1. Deploy the dashboard
2. Check that the links have been updated

## Test Impact

No current testing of ISV or documentation

## Request review criteria:

Self checklist (all need to be checked):
- [X] The developer has manually tested the changes and verified that the changes work
- [x] Commits have been squashed into descriptive, self-contained units of work (e.g. 'WIP' and 'Implements feedback' style messages have been removed)
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Review by manaswinidas — CHANGES_REQUESTED** [review]

File: manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml [hunk 1]
```diff
@@ -6,65 +6,50 @@ metadata:
...
+        ### Prerequisites
+        - You have logged in to Open Data Hub.
+        - You know the names and values that you want to use for environment variables in your notebook server environment, for example, `AWS_SECRET_ACCESS_KEY`.
+        - If you want to work with a very large data set, work with your administrator to proactively increase the storage capacity of your notebook server.
+        ### Launch a notebook
+        1. On the **Enabled applications** page, locate the **Jupyter** card.
+        2. Click **Launch application**.
+           a) If prompted, select your identity provider.
+           b) Enter your credentials and click **Log in** (or equivalent for your identity provider).
+           c) If you see an **Access permission needed** message, you are not in the default user group or the default administrator group for OpenShift Data Hub. Contact your administrator so that they can add you to the correct group.
```
Comment (manaswinidas) [inline manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml:32]: Is this a typo? `OpenShift Data Hub`

Comment (MelissaFlinn) [reply manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml:32]: Yes - it is a typo :(
It should be `Open Data Hub`

Comment (manaswinidas) [reply manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml:32]: This typo is present twice. So I got confused 😅

Comment (MelissaFlinn) [reply manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml:32]: ugh - thank you for catching these typos and sorry for the confusion

Comment (lucferbux) [reply manifests/apps/jupyter/create-jupyter-notebook-quickstart.yaml:32]: Let me update this ^^

File: manifests/apps/jupyter/deploy-python-model-quickstart.yaml [hunk 2]
```diff
@@ -6,64 +6,64 @@ metadata:
     opendatahub.io/categories: 'Model serving,Getting started'
 spec:
   displayName: Deploying a sample Python application using Flask and OpenShift.
-  appName: jupyterhub
```
Comment (lucferbux) [inline manifests/apps/jupyter/deploy-python-model-quickstart.yaml:9]: Now that I'm reviewing this @MelissaFlinn @andrewballantyne @DaoDaoNoCode @shalberd  we as Red Hat, are no longer promoting s2i and we are moving away all our tech stack. 
Does it make sense to keep this tutorial? I would say that  we might need to deprecate it.

Comment (DaoDaoNoCode) [reply manifests/apps/jupyter/deploy-python-model-quickstart.yaml:9]: I think we can keep it for now. We can always remove it as needed later.

**Review by DaoDaoNoCode — COMMENTED** [review]
/lgtm

**Review by manaswinidas — CHANGES_REQUESTED** [review]

File: manifests/apps/jupyter/deploy-python-model-quickstart.yaml [hunk 3]
```diff
@@ -6,64 +6,64 @@ metadata:
...
+        2. Click **Use this template**.
+        3. Type a *Repository name* for your new repository.
+        4. Select the **Public** radio button to ensure that the repository is visible.
+        5. Click **Create repository from template**.
       review:
         instructions: |-
-          #### To verify you have create a new repository:
-          Is a new GitHub repository visible with the name you entered?
+          #### Verify that you created a new repository:
+          In Github's **Your repositories** section, do you see the repository that you created from the s2i Flask template?
```
[the remark this answers is not in the document]
Comment (lucferbux) [reply manifests/apps/jupyter/deploy-python-model-quickstart.yaml]: Oh weird, didn't see this, thanks, updated!

**Review by DaoDaoNoCode — APPROVED** [review]
lgtm!

**Comment by shalberd** [comment]
very good work, thank you all so much. As discussed with @andrewballantyne on Slack, it'd be nice if this could find its way into incubation.

**Pull request merged**